src/CGT.jl

```diff
@@ -90,11 +89,11 @@ end
 
 function compute(sales::Sales{Currency{:EUR}}; details=false)
 	total_gain = by_period(sales; details)
-	net_gain = total_gain >= Allowance ? total_gain - Allowance : total_gain
-	tax = max(net_gain, 0EUR) * Tax
+	taxable_gain = max( total_gain - Allowance, 0EUR )
```

Comment:
I think the right fix here is
net_gain = total_gain < 0EUR ? total_gain : 
                   total_gain > Allowance ? total_gain - Allowance : 
                   0
This is because we want to show negative gains (losses) as well so people can make a note of losses to carry forward.

Reply:
The actual net gain should be `total_gain - Tax * taxable_gain`. This is why I made that distinction between net and taxable.